kitsune/sumo/static/sumo/js/aaq.js

```diff
@@ -14,16 +14,17 @@
   AAQSystemInfo.prototype = {
     init: function($form) {
       var self = this,
-      $input;
+        $input;
 
-      // Autofill the user agent via js
+      // Autofill the user agent
       $form.find('input[name="useragent"]').val(navigator.userAgent);
 
       // Only guess at OS, FF version, plugins if we are on the desktop
       // asking a firefox desktop question (or s/desktop/mobile/).
       if ((BrowserDetect.browser === 'fx' && self.isDesktopFF()) ||
       (BrowserDetect.browser === 'm' && self.isMobileFF()) ||
-      (BrowserDetect.browser === 'fxos' && self.isFirefoxOS())) {
+      (BrowserDetect.browser === 'fxos' && self.isFirefoxOS()) ||
+      (BrowserDetect.browser === 'fxios' && self.isFirefoxIOS())) {
```

Comment:
The first thing I typed was `FirefoxiOS`, and that was even weirder. I couldn't think of anything better. `FirefoxForIOS` sounds like a good one though.